pkg/daemon/update.go

```diff
@@ -419,6 +419,12 @@ func calculatePostConfigChangeActionFromFileDiffs(diffFileSet []string) (actions
 		imageRegistryAuthFile,
 		"/var/lib/kubelet/config.json",
 	}
+	directoriesPostConfigChangeActionNone := []string{
```

Comment:
Just as a note we're trying to move from a hard-coded list to an API here: https://github.com/openshift/api/pull/1764

So directories isn't part of our initially supported plans but we should eventually add support for this cc @djoshy